### PR TITLE
Factor our feed source model

### PR DESCRIPTION
**Pull request by estrattonbailey**

### CODE REVIEW

File: src/state/queries/like.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+export function useUnlikeMutation() {
+  const {agent} = useSession()
+
+  return useMutation({
+    mutationFn: async ({uri}: {uri: string}) => {
+      await agent.deleteLike(uri)
+    },
+  })
+}
```
Comment (estrattonbailey) [inline src/state/queries/like.ts:24]: Figured some generic helpers might be good?

File: src/state/queries/suggested-feeds.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    InfiniteData<AppBskyFeedGetSuggestedFeeds.OutputSchema>,
+    QueryKey,
+    string | undefined
+  >({
+    queryKey: suggestedFeedsQueryKey,
+    queryFn: async ({pageParam}) => {
+      const res = await agent.app.bsky.feed.getSuggestedFeeds({
+        limit: 10,
+        cursor: pageParam,
+      })
```
Comment (estrattonbailey) [inline src/state/queries/suggested-feeds.ts:23]: We only return 10 right now regardless, but in the future we may actually paginate

File: src/view/com/feeds/FeedSourceCard.tsx [hunk 3]
```diff
@@ -23,7 +22,7 @@ import {
 } from '#/state/queries/preferences'
 import {useFeedSourceInfoQuery} from '#/state/queries/feed'
 
-export const NewFeedSourceCard = observer(function FeedSourceCardImpl({
+export const FeedSourceCard = observer(function FeedSourceCardImpl({
```
Comment (estrattonbailey) [inline src/view/com/feeds/FeedSourceCard.tsx:25]: Had built a new version alongside the old. Replaced last old version with new and renamed.

File: src/view/com/util/post-embeds/index.tsx [hunk 4]
```diff
@@ -72,7 +72,13 @@ export function PostEmbeds({
     // custom feed embed (i.e. generator view)
     // =
     if (AppBskyFeedDefs.isGeneratorView(embed.record)) {
-      return <CustomFeedEmbed record={embed.record} />
+      return (
+        <FeedSourceCard
+          feedUri={embed.record.uri}
+          style={[pal.view, pal.border, styles.customFeedOuter]}
+          showLikes
+        />
```
Comment (estrattonbailey) [inline src/view/com/util/post-embeds/index.tsx:80]: Portable, only needs a URI

**Review by pfrazee — APPROVED** [review]
Lookin great

**Pull request merged**